lmdeploy/pytorch/models/llama.py

```diff
@@ -229,6 +230,199 @@ def forward(
         )
 
 
+class LlamaAttentionAscend(nn.Module):
+    """Rewrite module of LlamaAttention."""
+
+    def _load_weights(self, loader, rank: int, world_size: int,
```

Comment:
It is the same with `LlamaAttention::_load_weights`

Reply:
I once tried to use `LlamaAttentionAscend` to inherit `LlamaAttention` in order to reuse the `forward `function, but in fact, I used the `forward` function from `transformers.models.llama.modeling_llama.LlamaSdpaAttention`. @grimoire  told me to write it this way for now and he will adjust the inheritance later, so I wrote it this way for now.